sdk/core/azure-core/azure/core/polling/poller.py

```diff
@@ -172,6 +174,7 @@ def status(self):
         """
         return self._polling_method.status()
 
+    @distributed_trace
```

Comment:
I thought wait was enough?

Reply:
Yes so the SDK customer calls the result and when looking at the trace, it think its useful to see both otherwise you don't see how the wait and the resource are connected. Maybe @johanste has some opinions about this as well.


![lro example](https://user-images.githubusercontent.com/13604256/61743332-947cb580-ad84-11e9-865e-b5f989e1d75c.png)

Reply:
alright. @Johan says we should only have wait and the network call. Will get rid of it. :)